core/dbt/cli/params.py

```diff
@@ -217,19 +217,17 @@
 profiles_dir = click.option(
     "--profiles-dir",
     envvar="DBT_PROFILES_DIR",
-    help=f"Which directory to look in for the profiles.yml file. Default = {PurePath.joinpath(Path.home(), '.dbt')}",
-    default=PurePath.joinpath(Path.home(), ".dbt"),
-    type=click.Path(
-        exists=True,
-    ),
+    help="Which directory to look in for the profiles.yml file. If not set, dbt will look in the current working directory first, then HOME/.dbt/",
+    default=None,
```

Comment:
@dbeatty10 This isn't going to get you the expected behavior.  Let's talk about what you're looking for here, but as a headsup I'm going to be reverting these changes in a PR today..

Reply:
Thanks for the heads-up @iknox-fa 👍
I'll reach out to you discuss the ins-outs of our new Click CLI.